include/nbl/video/IPhysicalDevice.h

```diff
@@ -26,6 +26,8 @@ class IPhysicalDevice : public core::Interface, public core::Unmovable
     public:
         struct SLimits
         {
+            uint32_t apiVersion;
```

Comment:
this is specific to the backend, shouldn't be reported here.

Reply:
I think what you really wanted to expose is `spirvVersionSupport` not the API specific version of the API

Reply:
Yeah much better

Reply:
Resolved